db/migrate/20160203130657_add_rejection_message_to_trip.rb

```diff
@@ -1,5 +1,5 @@
 class AddRejectionMessageToTrip < ActiveRecord::Migration
   def change
-    add_column :trips, :rejection_message, :text
+  	add_column :trips, :rejection_message, :text, default: ''
```

Comment:
Aber sollte das dann nicht trotzdem in eine neue Migration? Vielleicht liege ich auch falsch, aber führt diese Änderung einer vorhandenen Migration nicht dazu, dass die Migration bei allen anderen, die den vorherigen Stand hatten, nicht mehr abgearbeitet wird und es so zu DB-Problemen kommt?